### PR TITLE
Visit Filter on query and archive thumbnails pages

### PR DESCRIPTION
**Pull request by BradleySappington**
Add new "visit" filter to thumbnail pages on archive and query result

### CODE REVIEW

**Comment by BradleySappington** [comment]
Screenshot 2023-12-14 at 12 26 59 PM

**Comment by BradleySappington** [comment]
@mfixstsci @bhilbert4 - Ready for review

**Review by mfixstsci — APPROVED** [review]
Nice, looks like adding the visit selection was fairly straight forward. Thanks for the submission @BradleySappington!

File: jwql/website/apps/jwql/data_containers.py [hunk 1]
```diff
@@ -2079,10 +2085,13 @@ def thumbnails_query_ajax(rootnames):
                    rootname in list(data_dict['file_data'].keys())]
     proposals = [data_dict['file_data'][rootname]['filename_dict']['program_id'] for
                  rootname in list(data_dict['file_data'].keys())]
+    visits = [data_dict['file_data'][rootname]['filename_dict']['visit'] for
+              rootname in list(data_dict['file_data'].keys())]
 
     dropdown_menus = {'instrument': instruments,
-                      'detector': detectors,
-                      'proposal': proposals}
+                      'detector': sorted(detectors),
```
Comment (bhilbert4) [inline jwql/website/apps/jwql/data_containers.py:2092]: I love sorting!

**Review by bhilbert4 — APPROVED** [review]
Looks good to me! Thanks!

**Pull request merged**